src/components/fields/text-field/text-field.tsx

```diff
@@ -91,12 +95,30 @@ export const TextField = (props: IGenericFieldProps<ITextFieldSchema | IEmailFie
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [value]);
 
+	useLayoutEffect(() => {
```

Comment:
avoid `useLayoutEffect` because it will throw a warning on nextjs

Reply:
removed, using useEffect instead